### DISCUSSION TITLE
Ability to migrate to a dual-stack cluster

### DISCUSSION THREAD

**Discussion by tearfulDalvik** [rancher/rke2#5162] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**

The default is ipv4 single stack cluster. I wonder, without changing the exisiting ipv4 podcidr, if there's a way to add ipv6 block to podcidr in order to migrate to a dual-stack cluster

**Describe the solution you'd like**
A solution like editing the `/etc/rancher/rke2/config.yaml`, and restart rke2-server

**Comment by manuelbuil** [comment, marked as answer]
Currently, there is no way to do this. The first component that should allow this is the CNI plugin and, as far as I know, neither Calico nor Cilium support it

**Reply by tearfulDalvik** [reply]
Thanks
